src/store/actions/createComment.js

```diff
@@ -0,0 +1,31 @@
+import { CREATE_COMMENT } from 'store/constants'
```

Comment:
lets chat about the strategy for actions moving forward. im not totally convinced yet about moving everything into actions... probably better but worth a chat

Reply:
For sure. This is a gray area, but I also fairly strictly move redux things out of a local store file if they are operating on the store outside of the context of that local store AND they are used outside of that component at least once. The more general thing going on here though is I'm trying to surface all the redux and graphql in one place to simplify later transitions away from the particular use of either. Happy to discuss. Needless to say I've a lot of things in motion in me from being in this code for 3 years and some of them you may find value in, some of them you may wish to go a different way with...